include/nccl_ofi_param.h

```diff
@@ -165,6 +165,13 @@ OFI_NCCL_PARAM_STR(topo_file_template, "TOPO_FILE_TEMPLATE", NULL);
  */
 OFI_NCCL_PARAM_INT(disable_native_rdma_check, "DISABLE_NATIVE_RDMA_CHECK", 0);
 
+/*
+ * Disable the check for required GDR support on EC2 instances. When this check
+ * is disabled, the plugin can be used without GDR support even on platforms
+ * that support GDR (P4d and later). By default, the plugin performs the check.
+ */
+OFI_NCCL_PARAM_INT(disable_required_gdr_check, "DISABLE_REQUIRED_GDR_CHECK", 0);
```

Comment:
GDR_REQUIRED.  This sounds like disabling a required check for GDR, not disabling the check if gdr should be required.